src/components/PopupEditAvatar.js

```diff
@@ -0,0 +1,38 @@
+import Popup from "./Popup.js";
+
+class PopupEditAvatar extends Popup {
```

Comment:
You should use `PopupWithForm` class for handling the avatar form, there is no need to create a new class